### DISCUSSION TITLE
Any plans to use System.Numerics types?

### DISCUSSION THREAD

**Discussion by amerkoleci** [notgiven688/jitterphysics2#195] [Ideas]
Hi,
Great job with this physics engine, question, I saw you're using own JVector, JQuaternion and JMatrix (which seams Matrix3x3), any plans to use Vector3, Quaternion and Matrix4x4 from System.Numerics?

Thanks in advance,
Amer

**Comment by notgiven688** [comment]
Hi @amerkoleci,

> Great job with this physics engine

Thank you.

> , I saw you're using own JVector, JQuaternion and JMatrix (which seams Matrix3x3), any plans to use Vector3, Quaternion and Matrix4x4 from System.Numerics?

No plans.

**Comment by notgiven688** [comment]
There are implicit conversion operators from JQuaternion and JVector to System.Numerics.Vector3 and System.Numerics.Quaternion.
